modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -71,6 +71,7 @@
       request_headers     => ['set X-Forwarded-Proto https'],
 
       rewrites            => [
+        {rewrite_rule => '^/grafana/(.*)$ http://127.0.0.1:8990/$1 [P]'},
```

Comment:
Why not use `grafana.ocf.berkeley.edu` as a separate vhost instead of having this as a path under prometheus?

Reply:
i agree here. it seems odd to have grafana under the prometheus vhost when prometheus is grafana's data source. the other way around would make more sense, but separate vhosts is better